### PR TITLE
Summary, usage, version labels for sanity_linters

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This PR brings tests for "summary", "usage", "version" labels.

### CODE REVIEW

File: colin/checks/labels/summary.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from colin.checks.abstract.labels import LabelCheck
+
+
+class SummaryCheck(LabelCheck):
+
+    def __init__(self):
+        super().__init__(name="summary_label_required",
+                         message="Label 'summary' has to be specified.",
+                         description="A short description of the image.",
+                         reference_url="http://url-to-upstream-documentation/summary",
```
Comment (lachmanfrantisek) [inline colin/checks/labels/summary.py]: Do we have some real url for all of these checks?

File: colin/checks/labels/usage.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from colin.checks.abstract.labels import LabelCheck
+
+
+class UsageCheck(LabelCheck):
+
+    def __init__(self):
+        super().__init__(name="usage_label_required",
+                         message="Label 'usage' has to be specified.",
+                         description="A human readable example of container execution .",
```
Comment (lachmanfrantisek) [inline colin/checks/labels/usage.py]: s/execution ./execution.

File: colin/checks/labels/summary.py (hunk 1 above)
Comment (phracek) [reply colin/checks/labels/summary.py]: What about this one https://fedoraproject.org/wiki/Container:Guidelines#LABELS ?

Comment (lachmanfrantisek) [reply colin/checks/labels/summary.py]: If the fedora-specific document is ok, then we can add this to all of the `label` checks.

Comment (TomasTomecek) [reply colin/checks/labels/summary.py]: definitely good enough (for now)

**Pull request merged**